### ISSUE TITLE
Clone row functionality does not show enum fields properly in UI

### ISSUE THREAD

**Issue by harishnair96** [hasura/graphql-engine#9254]
### Version Information

Server Version: 2.15.2

### Environment

OSS

### What is the current behaviour?

When clicking on clone row button enum fields are not shown properly in the UI. It just shows '-- enum value --'.
<img width="570" alt="Screenshot 2022-11-29 at 10 12 35 PM" src="https://user-images.githubusercontent.com/20351766/204590004-31997e1f-bf0a-47ec-944c-72fd7f0170ba.png">

### What is the expected behaviour?

Show value for enum fields as well just like for any other fields.
<img width="570" alt="Screenshot 2022-11-29 at 10 12 42 PM" src="https://user-images.githubusercontent.com/20351766/204590110-73c2e022-86d9-4221-9cf7-a2f833eac2bb.png">

### How to reproduce the issue?

1. Run the following SQL 
  ```sql
  CREATE TABLE role (role TEXT PRIMARY KEY);
  INSERT INTO role VALUES ('user'), ('admin');
  CREATE TABLE person (id INTEGER PRIMARY KEY, name TEXT, role TEXT REFERENCES role (role));
  INSERT INTO person VALUES (1, 'John Doe', 'user');
  ```
2. Track _person_ and _role_ tables and mark _role_ table as enum
3. Click on 'Clone row' button in the _person_ table and observe the value shown for the _role_ field

### Keywords

clone row

**Comment by lucarestagno** [comment]
Hi @harishnair96 thanks for reporting the issue 🙌

We're planning to release the fix in the next release 2.16 that's planned to be released next week.

We hope it works well for you.

**Comment by dsandip** [comment]
https://hasura.io/changelog/community-edition/v2.16.0
